### PR TITLE
Using default framelength (Fixed IOS Crashes)

### PR DESCRIPTION
**Pull request by DeepakM02**
1. Prevent app crashing
2. Using default frame length

### CODE REVIEW

File: ios/Voice/Voice.m [hunk 1]
```diff
@@ -272,6 +273,12 @@ - (void) setupAndStartRecognizing:(NSString*)localeStr {
         [self teardown];
         return;
     }
+    }
+    @catch (NSException *exception) {
+    NSLog(@"[Error] - %@ %@", exception.name, exception.reason);
```
Comment (safaiyeh) [inline ios/Voice/Voice.m:278]: Can you remove this log

File: ios/Voice/Voice.m [hunk 2]
```diff
@@ -224,7 +225,7 @@ - (void) setupAndStartRecognizing:(NSString*)localeStr {
         [mixer installTapOnBus:0 bufferSize:1024 format:recordingFormat block:^(AVAudioPCMBuffer * _Nonnull buffer, AVAudioTime * _Nonnull when) {
             //Volume Level Metering
             //Buffer frame can be reduced, if you need more output values
-            [buffer setFrameLength: buffer.frameCapacity];
+            // [buffer setFrameLength: buffer.frameCapacity];
```
Comment (safaiyeh) [inline ios/Voice/Voice.m:228]: Should we make this a prop? so people can customize

**Comment by safaiyeh** [comment]
Thanks @DeepakM02 left a couple comments

File: ios/Voice/Voice.m (hunk 2 above)
Comment (DeepakM02) [reply ios/Voice/Voice.m:228]: Do we need to specify frameCapacity?

File: ios/Voice/Voice.m (hunk 1 above)
Comment (DeepakM02) [reply ios/Voice/Voice.m]: Done

File: ios/Voice/Voice.m (hunk 2 above)
Comment (safaiyeh) [reply ios/Voice/Voice.m:228]: Gotcha, checked the previous commits looks like we don't need it. Lets clean up the code and delete the line & comments

**Comment by DeepakM02** [comment]
@safaiyeh you can review this

**Pull request merged**